src/control/server/storage/scm/ipmctl.go

```diff
@@ -346,36 +351,63 @@ func parseShowRegionOutput(log logging.Logger, text string) (_ storage.ScmState,
 			if !appDirect {
 				continue
 			}
-			c, err := humanize.ParseBytes(kv[1])
+			fc, err := humanize.ParseBytes(kv[1])
 			if err != nil {
-				return storage.ScmStateUnknown, err
+				return nil, err
 			}
-			totalFree += c
+			regionFreeBytes = append(regionFreeBytes, fc)
 		}
 		appDirect = false
 	}
 
-	if totalFree > 0 {
-		log.Debugf("PMem regions have %s free capacity", humanize.Bytes(totalFree))
-		return storage.ScmStateFreeCapacity, nil
-	}
-	log.Debug("PMem regions have no free capacity")
-	return storage.ScmStateNoFreeCapacity, nil
+	return regionFreeBytes, nil
 }
 
-func (cr *cmdRunner) getRegionState() (storage.ScmState, error) {
+func (cr *cmdRunner) getRegionFreeSpace() ([]uint64, error) {
 	out, err := cr.showRegions()
 	if err != nil {
-		return storage.ScmStateUnknown, errors.WithMessage(err, "show regions cmd")
+		return nil, errors.WithMessage(err, "show regions cmd")
 	}
 
 	cr.log.Debugf("show region output: %s\n", out)
 
 	if strings.Contains(out, outScmNoRegions) {
+		return []uint64{}, nil
```

Comment:
nit: A `nil` is equivalent to an empty slice, as far as the caller is concerned.

Reply:
done